src/main/resources/alma/maps.xml

```diff
@@ -4,7 +4,7 @@
   <xi:include href="maps/alma-institution-code-to-isil.xml" />
   <xi:include href="maps/alma-iz-code-to-isil.xml" />
   <xi:include href="maps/rswk-indicator.xml" />
-
+	<filemap name="deweyLabels" files="src/main/resources/alma/maps/deweyLabels.tsv"/>
```

Comment:
don't reference it like this. Also, we only need the test file. Analog to https://github.com/hbz/lobid-resources/blob/master/src/main/resources/morph-hbz01-to-lobid.xml#L7685 , should work.

Reply:
@dr0i Thisdoes not work, i tried it like this before. The reason is, that in  https://github.com/hbz/lobid-resources/blob/master/src/main/resources/morph-hbz01-to-lobid.xml#L7685 the deweyLabel file is at the same level als the morph. We need to transfer the  map file at least in the maps folder as the other maps..